### PR TITLE
Apply transition time

### PR DESCRIPTION
**Pull request by PritimoySarkar**
Applying transition time in the chat list expansion transition so the animation will look more soothing to eyes

### CODE REVIEW

**Comment by PritimoySarkar** [comment]
just one line to apply **transition time** in the chat list expansion transition so the animation will look more soothing to eyes

File: messenger-hover.css [hunk 1]
```diff
@@ -43,6 +43,7 @@
     flex-basis: 25%;
     max-width: 420px;
     min-width: 320px;
+    transition 0.7s;
```
Comment (ekohilas) [inline messenger-hover.css:46]: There is a missing colon after transition.

**Comment by ekohilas** [comment]
If you want to add this feature in the future, I'm happy to add it if it's set up as an option, as personally I find the 0.7s transition time too long for this kind of menu.

**Comment by PritimoySarkar** [comment]
yaa I get it.
But unfortunately I don't know how to add it as an option so it's alright
I'll try to add it as an option later.

thank you for your response

**Pull request closed without merging**